README.md

```diff
@@ -64,6 +65,20 @@ Gather feedback where uncertainty exists. We are our own best testing team and a
 - **Move fast**: Putting feedback loops in place early on helps us make more informed decisions collaboratively and commit to deliverables that create tangible progress.
 - **Risk management**: Even with great design and careful planning, unplanned work is guaranteed to appear once you're in the process of building. So better start early. 
 
+## Cult of Quality
+
+Our teams appear cult-like when it comes to quality. We believe the code we produce is always our best work and represents our level of professional competence. No excuses.
```

Comment:
"We believe the code we produce is always our best work _for the exisiting context_" ? maybe somethinkg like this could scope the changes to not fall in the "perfectionism" thread?

Reply:
I’m really not a fan of weakening the principle of demanding ourselves to do our best work by adding a cop-out like ”depending on external circumstances” or ”for the existing context”.

It’s just too easy to read as permission to not take pride in your work due to some circumstance like time constraints or unexpected problems. 👈 These will always happen anyways!

I understand this can be confused with perfectionism but it really isn’t. This is pragmatism.

Of course context matters but we should never accept shipping something we believe is not our best work. The pragmatic approach is to find a great solution given a set of parameters. No, it’s never perfect but it should be us working at our highest level => our best work.